src/main/java/com/adobe/epubcheck/tool/EpubChecker.java

```diff
@@ -62,6 +62,13 @@
 public class EpubChecker
 {
 
+  static {
+    /* fix #665 (window-less "Checker" gui app on Mac)
+     * set -Djava.awt.headless=true programmatically as early as possible
+     */
+    System.setProperty("java.awt.headless", "true");
```

Comment:
Could also have been added to the static main method in `Checker`, but as far as I can tell this is OK too.